testsuite/tests/conftest.py

```
@@ -95,6 +96,21 @@ def pytest_report_header(config):
     return header
 
 
+@pytest.hookimpl(trylast=True)
+def pytest_sessionfinish(session, exitstatus):  # pylint: disable=unused-argument
+    """Pretty-print JUnit XML files after test session"""
+    junit_xml = session.config.option.xmlpath
+    if junit_xml and os.path.exists(junit_xml):
+        try:
+            dom = xml.dom.minidom.parse(junit_xml)
```

Ruff (0.15.9)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S318)